src/actions/users.js

```diff
@@ -1397,6 +1397,16 @@ export function clearUserAccessTokens(): ActionFunc {
     };
 }
 
+export function checkForModifiedUsers() {
+    return (dispatch: DispatchFunc, getState: GetStateFunc) => {
+        const state = getState();
+        const users = getUsers(state);
+        const lastDisconnectAt = state.websocket.lastDisconnectAt;
+
+        return dispatch(getProfilesByIds(Object.keys(users), {since: lastDisconnectAt}));
```

Comment:
So on an older server, this will fetch all the users that you have already loaded in the store, right?

Reply:
Oh right, I forgot to add the version check. Thanks for the reminder